nexus/src/db/fixed_data/role_assignment_builtin.rs

```diff
@@ -42,5 +51,14 @@ lazy_static! {
                 *FLEET_ID,
                 role_builtin::FLEET_VIEWER.role_name,
             ),
+
+            // The "internal-authenticator" user gets the "authentiator" role on
```

Comment:
is this meant to say `external-authenticator`? also typo: "authentiator"

Reply:
Nice catch!